backend/core/src/auth/user.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { OmitType } from "@nestjs/swagger"
+import { User } from "../entity/user.entity"
+import { Expose } from "class-transformer"
+
+export class RegisterResponseDto extends OmitType(User, ["passwordHash", "applications"] as const) {
+  @Expose()
+  accesToken: string
```

Comment:
spelling - should be `accessToken`

Reply:
I agree we should fix this before merge.

Reply:
Fixed.